tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+  ldiag3d      = @[LDIAG3D]
+  qdiag3d      = @[QDIAG3D]
+  lightning_threat = @[LIGHTNING_THREAT]
+  print_diff_pgr = @[PRINT_DIFF_PGR]
+  fhcyc        = @[FHCYC]
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = @[IMP_PHYSICS]
+  iovr         = @[IOVR]
+  lcnorm       = @[LCNORM]
+  ltaerosol    = .true.
+  lradar       = .true.
+  ttendlim     = -999
+  dt_inner     = @[DT_INNER]
+  sedi_semi    = @[SEDI_SEMI]
+  decfl        = @[DECFL]
+  oz_phys      = @[OZ_PHYS_NEW]
+  oz_phys_2015 = @[OZ_PHYS_OLD]
+  lsoil_lsm    = @[LSOIL_LSM]
+  do_mynnedmf  = @[DO_MYNNEDMF]
+  do_mynnsfclay = @[DO_MYNNSFCLAY]
+  icloud_bl    = 1
+  bl_mynn_edmf = 1
+  bl_mynn_tkeadvect = @[BL_MYNN_TKEADVECT]
+  bl_mynn_edmf_mom = 1
+  do_ugwp      = @[DO_UGWP]
+  do_tofd      = @[DO_TOFD]
+  do_ca        = @[DO_CA]
+  gwd_opt      = @[GWD_OPT]
+  do_ugwp_v0   = @[DO_UGWP_V0]
+  do_ugwp_v1   = @[DO_UGWP_V1]
+  do_ugwp_v0_orog_only = @[DO_UGWP_V0_OROG_ONLY]
+  do_ugwp_v0_nst_only  = @[DO_UGWP_V0_NST_ONLY]
+  do_gsl_drag_ls_bl    = @[DO_GSL_DRAG_LS_BL]
+  do_gsl_drag_ss       = @[DO_GSL_DRAG_SS]
+  do_gsl_drag_tofd     = @[DO_GSL_DRAG_TOFD]
+  do_ugwp_v1_orog_only = @[DO_UGWP_V1_OROG_ONLY]
+  do_ugwp_v1_w_gsldrag = @[DO_UGWP_V1_W_GSLDRAG]
+  min_lakeice  = @[MIN_LAKEICE]
+  min_seaice   = @[MIN_SEAICE]
+  use_cice_alb = @[USE_CICE_ALB]
+  pdfcld       = .false.
+  fhswr        = @[FHSWR]
+  fhlwr        = @[FHLWR]
+  ialb         = @[IALB]
+  iems         = @[IEMS]
+  iaer         = @[IAER]
+  icliq_sw     = @[ICLIQ_SW]
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  elmx         = @[ELMX]
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = @[CNVGWD]
+  cpl_imp_mrg  = @[CPL_IMP_MRG]
+  shal_cnv     = @[SHAL_CNV]
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = @[HYBEDMF]
+  satmedmf     = @[SATMEDMF]
+  isatmedmf    = @[ISATMEDMF]
+  lheatstrg    = @[LHEATSTRG]
+  lseaspray    = @[LSEASPRAY]
+  random_clds  = @[RANDOM_CLDS]
+  trans_trac   = .true.
+  cnvcld       = @[CNVCLD]
+  imfshalcnv   = @[IMFSHALCNV]
+  imfdeepcnv   = @[IMFDEEPCNV]
+  do_deep      = @[DO_DEEP]
+  progsigma    = @[PROGSIGMA]
+  betascu      = @[BETASCU]
+  betamcu      = @[BETAMCU]
+  betadcu      = @[BETADCU]
+  betal_deep   = @[BETAL_DEEP]
+  betas_deep   = @[BETAS_DEEP]
+  ras          = @[RAS]
+  cdmbgwd      = @[CDMBWD_NEST02]
+  clam_deep    = @[CLAM_DEEP]
+  prslrd0      = 0.
+  ivegsrc      = 1
+  isot         = 1
+  lsoil        = 4
+  lsm          = @[LSM]
+  iopt_dveg    = @[IOPT_DVEG]
+  iopt_crs     = @[IOPT_CRS]
+  iopt_btr     = 1
+  iopt_run     = 1
+  iopt_sfc     = @[IOPT_SFC]
+  iopt_frz     = 1
+  iopt_inf     = 1
+  iopt_rad     = @[IOPT_RAD]
+  iopt_alb     = @[IOPT_ALB]
+  iopt_snf     = 4
+  iopt_tbot    = 2
+  iopt_stc     = @[IOPT_STC]
+  iopt_trs     = @[IOPT_TRS]
+  iopt_diag    = @[IOPT_DIAG]
+  sfc_rlm      = @[SFC_RLM]
+  sfc_z0_type  = @[SFC_Z0_TYPE]
```

Comment:
the current global workflow does not have line 233, 234:
`sfc_rlm, sfc_z0_type`

Reply:
They are quite deprived.

Reply:
These variables are unneeded, so I removed them. (Removing them doesn't change the results.)